todo_app/data/trello_items.py

```diff
@@ -0,0 +1,194 @@
+        for card in card_list['cards']:
+            cards.append(Item.from_trello_card(card, card_list))
+
+    return cards
+
+
+def get_card(cardId):
+    """
+    Checks the card using the ID.
+    Args:
+        id (str): The ID of the card.
+    """
+    singleCard = []
+    cards = get_cards()
+
+    for card in cards:
+        if card.id == cardId:
+            singleCard.append(card)
+
+    return singleCard
+
+
+def add_card(cardName, cardDesc, cardDue):
+    """
+    Adds a new card with the specified name.
+
+    Args:
+        name: The name of the card.
+    """
+    todo_list = get_list('To Do')
+
+    params = build_params({ 'name': cardName, 'desc': cardDesc, 'due': cardDue, 'idList': todo_list['id'] })
+    url = build_url('/cards')
+
+    response = requests.post(url, params = params)
+    card = response.json()
+
+    return Item.from_trello_card(card, todo_list)
+
+
+def start_card(cardId):
+    """
+    Moves the card with the specified ID to the "In Progress" list.
+    Args:
+        id (str): The ID of the card.
+    Returns:
+        card: The saved card, or None if no cards match the specified ID.
+    """
+    in_progress_list = get_list('In Progress')
+    card = update_card_list(cardId, in_progress_list)
+
+    return Item.from_trello_card(card, in_progress_list)
+
+
+
+def complete_card(cardId):
+    """
+    Updates an existing card in the session. If no existing card matches the ID of the specified card, nothing is saved.
+
+    Args:
+        card: The card to save.
+    """
+    completed_list = get_list('Done')
+    card = update_card_list(cardId, completed_list)
+
+    return Item.from_trello_card(card, completed_list)
+
+
+def undo_card(cardId):
+    """
+    Moves the item with the specified ID to the "To-Do" list.
+    Args:
+        id (str): The ID of the card.
+    Returns:
+        card: The saved card, or None if no cards match the specified ID.
+    """
+    todo_list = get_list('To Do')
+    card = update_card_list(cardId, todo_list)
+
+    return Item.from_trello_card(card, todo_list)
+
+
+def archive_card(cardId):
+    """
+    Updates an existing card in the session. If no existing card matches the ID of the specified card, nothing is archived.
+
+    Args:
+        card: The card to archive.
+    """
+    completed_list = get_list('Done')
+    card = update_card_status(cardId, completed_list)
+
+    return Item.from_trello_card(card, completed_list)
+
+
+def update_card_list(cardId, list):
+    """
+    Runs a put request to update the card list
+    Args:
+        cardId: The ID of the card to be updated
+        list: The list object to use for the new list
+    Returns: 
+        card: The updated card, in the new specified list
+    """
+    params = build_params({ 'idList': list['id'] })
+    url = build_url('/cards/%s' % cardId)
+
+    response = requests.put(url, params = params)
+    card = response.json()
+
+    return card
+
+
+def update_card_status(cardId, list):
```

Comment:
Minor, but this function name is a little unclear, maybe `close_and_move_card`?

Reply:
Updated as requested.